src/NzbDrone.Core/MediaFiles/DiskScanService.cs

```diff
@@ -120,6 +120,9 @@ public void Scan(Movie movie)
             var videoFilesStopwatch = Stopwatch.StartNew();
             var mediaFileList = FilterFiles(movie, GetVideoFiles(movie.Path)).ToList();
 
+            if (_configService.DeleteEmptyFolders && mediaFileList.Count == 0)
```

Comment:
I think we should catch here if the user has enabled both options, issue a warning and do nothing.

Reply:
Also not sure if diskProvider automatically uses recycleBin, otherwise we should use recycleBin here instead.